packages/headless/doc-parser/src/tsdoc-emitter.ts

```diff
@@ -8,13 +8,13 @@ import {
 } from '@microsoft/tsdoc';
 
 export function emitAsTsDoc(nodes: readonly DocNode[]) {
-  return nodes.map(emitNode).join('');
+  return nodes.map(emitNode).join('').trimRight();
```

Comment:
Any reason to just trim the right side? Why not trim both sides?

Reply:
Oh, I was initially worried that trimming the left would affect plaintext nodes, but I just tried it out and it seems fine, so I'll push that change before merging 👍🏻